packages/Liquid-Tests.package/LQStatisticsWorkbenchIntegrationTests.class/instance/testStatisticsWorkbenchIsDefined.st

```diff
@@ -0,0 +1,6 @@
+as yet unclassified
+testStatisticsWorkbenchIsDefined
```

Comment:
I like this test! Makes it easy to see if SW was installed properly

Reply:
Yep, since we had some problems in the past with SW not being installed for some, I thought it would be nice to have a test for that, especially since this also makes sure SW is installed when the CI is running, meaning the setup in the baseline is correct.